Packages/vcs/Lib/VTKPlots.py

```diff
@@ -406,9 +406,11 @@ def portrait(self, W, H, x, y, clear):
             self.renWin.SetSize(W, H)
 
     def initialSize(self):
-        # screenSize = self.renWin.GetScreenSize()
-        self.renWin.SetSize(self.canvas.bgX, self.canvas.bgY)
-        self._lastSize = (self.canvas.bgX, self.canvas.bgY)
+        screenSize = self.renWin.GetScreenSize()
+        bgX = int(screenSize[0]*.6)
```

Comment:
what are these magic numbers? (0.6). It would be nice if you can add a comment here.